packages/block-library/src/image/image.js

```diff
@@ -118,33 +118,25 @@ export default function Image( {
 			const {
 				getBlock: _getBlock,
 				getBlockRootClientId,
-				getBlockTransformItems,
 				getSettings,
+				canInsertBlockType,
 			} = select( blockEditorStore );
 
-			const block = _getBlock( clientId );
 			const rootClientId = getBlockRootClientId( clientId );
-			const transformations = getBlockTransformItems(
-				[ block ],
-				rootClientId
-			);
```

Comment:
Both getBlock and getBlockTransformItems are slow selectors and should be avoided as much as we can.